src/lib/services/index.ts

```diff
@@ -137,6 +139,17 @@ export const createServices = (
         settingService,
     );
 
+    const schedulerService = new SchedulerService();
+    schedulerService.schedule(
+        apiTokenService.fetchActiveTokens,
+        minutesToMilliseconds(1),
+    );
+
+    schedulerService.schedule(
+        apiTokenService.updateLastSeen,
+        minutesToMilliseconds(3),
+    );
+
```

Comment:
when we move to package by feature I would think we could have a pattern where we have a separate file `scheduled-jobs.ts` to deal with the registering these things.

Reply:
definitely, that's the plan. I can't wait to split everything by feature :)